pulp_maven/tests/functional/utils.py

```diff
@@ -32,3 +34,13 @@ async def _download_file(url, auth=None, headers=None):
     async with aiohttp.ClientSession(auth=auth, raise_for_status=True) as session:
         async with session.get(url, verify_ssl=False, headers=headers) as response:
             return Download(body=await response.read(), response_obj=response)
+
+
+def generate_jar(full_path, size=1024, relative_path=None):
+    """Generate a random file."""
```

Comment:
I guess, you will keep the possibility for switching to an actual "jar" file eventually?